src/__a11y__/a11y-page-object.ts

```diff
@@ -113,7 +113,7 @@ function ariaLevelViolationsFilter(violation: Axe.Result) {
 // There is a known issue when multiple app layout instances rendered on the page. Skip them for pages matching this pattern
 function landmarkViolationFilter(violation: Axe.Result, currentUrl: string) {
   return (
-    !currentUrl.includes('app-layout/multi-layout-simple') ||
+    !currentUrl.includes('app-layout/multi') ||
```

Comment:
This is a known issue that multiple app layouts cause this landmark violation. Extended the pattern to also cover the new pages